src/main/drivers/serial_uart_impl.h

```diff
@@ -29,7 +29,7 @@
 #define UART_RX_BUFFER_SIZE     128
 #endif
 #ifndef UART_TX_BUFFER_SIZE
-#define UART_TX_BUFFER_SIZE     256
+#define UART_TX_BUFFER_SIZE     1024
```

Comment:
Not sure if there is much point in changing this for unsupported legacy targets - this might just break them if they don't have enough RAM available.